go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/gorilla/websocket v1.5.3
 	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
+	github.com/jlaffaye/ftp v0.2.0
```

Comment:
this import is not used in gofr, should be removed from here, as it is already present in ftp package